Doc/library/collections.abc.rst

```diff
@@ -106,7 +106,7 @@ a :class:`Mapping`.
 
 .. versionadded:: 3.9
    These abstract classes now support ``[]``. See :ref:`types-genericalias`
-   and :pep:`585`.
+   and :pep:`585`. They, however, do not inherit from :class:`~typing.Generic` explicitly.
```

Comment:
Is this added note necessary? The existing `types-genericalias` link takes you to https://docs.python.org/3/library/stdtypes.html#types-genericalias, which I think makes it clear that they don't explicitly inherit from generics

Reply:
Indeed it's a remainder from a now reverted attempt to list generic arguments there - it can be removed now.